app/models/manageiq/providers/vmware/infra_manager/vm_or_template_shared/operations/relocation.rb

```diff
@@ -52,6 +52,12 @@ def raw_relocate(host, pool = nil, datastore = nil, disk_move_type = nil, transf
     pool_mor      = pool.ems_ref_obj      if pool
     datastore_mor = datastore.ems_ref_obj if datastore
 
+    disk_move_type = case disk_transform
+      when 'thin'  then VimString.new('sparse', "VirtualMachineRelocateTransformation")
+      when 'thick' then VimString.new('flat', "VirtualMachineRelocateTransformation")
+      else disk_transform
```

Comment:
I wanted to continue to handle existing queued methods that pass this argument in as a `VirtualMachineRelocateTransformation` type so if it isn't thin or thick we just use whatever was passed in for compatibility